.gitignore

```diff
@@ -1,6 +1,7 @@
 *.gem
 *.rbc
 *.swp
+*.idea
```

Comment:
I'd suggest that IDE-specific ignores should be set in a global `.gitignore` and not added to the repo.